ut/abnormal_column_names_test.cpp

```diff
@@ -0,0 +1,47 @@
+#include "abnormal_column_names_test.h"
+#include "utils.h"
+
+#include <clickhouse/columns/column.h>
+#include <clickhouse/block.h>
+#include <unordered_set>
+#include <iostream>
+
+namespace {
+    using namespace clickhouse;
+}
+
+void AbnormalColumnNamesTest::SetUp() {
+    client_ = std::make_unique<Client>(std::get<0>(GetParam()));
+}
+
+void AbnormalColumnNamesTest::TearDown() {
+    client_.reset();
+}
+
+// Sometimes gtest fails to detect that this test is instantiated elsewhere, suppress the error explicitly.
+GTEST_ALLOW_UNINSTANTIATED_PARAMETERIZED_TEST(AbnormalColumnNamesTest);
+TEST_P(AbnormalColumnNamesTest, Select) {
+
+    const auto & queries = std::get<1>(GetParam());
+    for (const auto & query : queries) {
+        std::unordered_set<std::string> names;
+        size_t count = 0;
+        client_->Select(query,
+            [& query,& names, & count](const Block& block) {
+                if (block.GetRowCount() == 0 || block.GetColumnCount() == 0)
+                    return;
+
+                std::cout << "query => " << query <<"\n" << PrettyPrintBlock{block}; 
```

Comment:
It doesn't make sense to just print a block to STDOUT for CI/CD, nobody is going to manually verify the validity of the output. Please update the test to have `ASSERT_`/`EXPECT_` statements that actually check something.